rvr-crud

rvspt opened the pull request:
Initial deploy of user create, read, update and delete history
analysis.

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### Description of the Change

<!--

1- Altered _governanceCollector/agent/lib/harvester.js_ and _governanceCollector/agent/lib/userAccessControl.js_ to include time of snapshot for apps and sheets user privileges collection.
2- Created new _installer/apps/User Create, Read, Update, Delete Privileges.qvf_ that plots users privileges evolution for Create; Update, Read and Delete privileges for both apps on Sense site and sheets within the apps available on the site.

-->

### Alternate Designs

Not applicable.

### Why Should This Be In Core?

Quickly understand how many users can read only or create contents on a site. I.e. user is create new applications or user is only allowed to create new sheets within applications.

### Benefits

Quickly understand how many users can read only or create contents on a site. I.e. user is create new applications or user is only allowed to create new sheets within applications.

### Possible Drawbacks

None detected.

### Applicable Issues

None detected.

rvspt committed:
Users CRUD Analysis v1

Initial deploy of user create, read, update and delete historic
analysis.

rvspt committed:
Removed unecessary comments

goldbergjeffrey reviewed (changes requested):
On Harvester.js, pull the snapshot_time code into a private function sitting down below.

make snapshot_time_formatted = 
``` snapshot_time_formatted = getSnapshotTime()```

private function
```function getSnapshotTime(){
//all the code for getting a new date and then //formatting the snapshottime.
return snapshottime
}

rvspt committed:
Private funtion for Snapshot Time

Created a private function to format the snapshot time of a CRUD harvest
for apps or sheets.

rvspt commented:
Done as requested
